### PR TITLE
[ADD] hr_expense_analytic_plans

### PR DESCRIPTION
**Pull request by hbrunn**
# Use analytic plans in expenses

This module allows you to assign analytic plans to expense lines instead of analytic accounts. This way, you can distribute the costs of an expense over multiple analytic accounts.

### CODE REVIEW

File: hr_expense_analytic_plans/tests/test_hr_expense_analytic_plans.py [hunk 1]
```diff
@@ -0,0 +1,63 @@
...
+        expense.signal_workflow('confirm')
+        expense.signal_workflow('validate')
+        expense.signal_workflow('done')
+        for line in expense.account_move_id.line_id:
+            if line.analytic_lines:
+                self.assertEqual(len(line.analytic_lines), 2)
+                self.assertIn(account1, [
+                    l.account_id for l in line.analytic_lines])
+                self.assertIn(account2, [
+                    l.account_id for l in line.analytic_lines])
```
Comment (ghost) [inline hr_expense_analytic_plans/tests/test_hr_expense_analytic_plans.py:65]: Can you add an amount and quantity to your expense line and assert that the amount is split 50% - 50% between your analytic lines please ?

**Comment by ghost** [comment]
@hbrunn LGTM except for one comment in your unit test

File: hr_expense_analytic_plans/__openerp__.py [hunk 2]
```diff
@@ -0,0 +1,45 @@
...
+#    GNU Affero General Public License for more details.
+#
+#    You should have received a copy of the GNU Affero General Public License
+#    along with this program.  If not, see <http://www.gnu.org/licenses/>.
+#
+##############################################################################
+{
+    "name": "Analytic plans in expenses",
+    "version": "1.0",
+    "author": "Therp BV",
```
Comment (adrienpeiffer) [inline hr_expense_analytic_plans/__openerp__.py]: Can you add OCA as author ? like `Odoo Community Association (OCA)`

File: hr_expense_analytic_plans/__openerp__.py [hunk 3]
```diff
@@ -0,0 +1,45 @@
...
+    ],
+    "test": [
+    ],
+    "auto_install": False,
+    "installable": True,
+    "application": False,
+    "external_dependencies": {
+        'python': [],
+    },
+}
```
Comment (adrienpeiffer) [inline hr_expense_analytic_plans/__openerp__.py:41]: Not mandatory but some keys aren't necessary

**Comment by adrienpeiffer** [comment]
Little remarks on the manifest. Otherwise :+1: (Code review and functionnal test)

**Comment by max3903** [comment]
Tested on runbot
:+1:

**Comment by mmalorni** [comment]
Tested on runbot. Work great! :+1:

**Comment by hbrunn** [comment]
@dufresnedavid @adrienpeiffer thanks for your hints, done

**Comment by mmalorni** [comment]
:+1:

**Comment by ghost** [comment]
@hbrunn Thanks! :+1:

**Pull request merged**